pavelib/quality.py

```diff
@@ -2,9 +2,12 @@
 Check code quality using pep8, pylint, and diff_quality.
 """
 from paver.easy import sh, task, cmdopts, needs, BuildFailure
+import json
 import os
 import re
 
+from openedx.core.djangolib.markup import HTML
```

Comment:
Reviewers: Is this an ok reference?  Note that this is in djangolib only because its tests make use of Mako.  But the library itself is just using markupsafe.  I could also have the linter ignore pavelib, which I did temporarily, but it was catching what could potentially be a real issue of breaking your HTML report, depending on what is appearing in the violation message.